karma.conf.js

```diff
@@ -88,8 +88,7 @@ module.exports = function(config) {
         files: [
             { pattern: 'test-context.js' },
             { pattern: 'test/files/*', included: false },
-            { pattern: 'node_modules/jquery/dist/jquery.js' },
-            { pattern: 'node_modules/sinon/pkg/sinon.js' }
+            { pattern: 'node_modules/jquery/dist/jquery.js' }
```

Comment:
Do we really need jquery.js embedded on the page that runs tests? I thought we import it into tests that use it (which means it's bundled in test-context.js).